.github/workflows/deploy.yaml

```diff
@@ -237,7 +237,7 @@ jobs:
 
       - name: pip_install_mda
         run: |
-          awk '/__version__ =/ {print $3; quit}' package/MDAnalysis/version.py | tr -d \" > version.dat
+          awk '/__version__ =/ {print $3; exit}' package/MDAnalysis/version.py | tr -d \" > version.dat
```

Comment:
```suggestion
          awk '/__version__ =/ {gsub("\"", "", $3); print $3; exit}' package/MDAnalysis/version.py > version.dat
```

Reply:
🤣 is this a really necessary change or are we just showing off `awk` knowledge at this point?

Reply:
I'm just showing off my age ;-p

Reply:
Ok, hard data:
```
time for x in seq 10000000; do awk '/__version__ =/ {gsub("\"", "", $3); print $3; exit}' package/MDAnalysis/version.py > version.dat; done

real	0m0.013s
user	0m0.002s
sys	0m0.005s
```

vs

```
time for x in seq 10000000; do awk '/__version__ =/ {print $3; exit}' package/MDAnalysis/version.py | tr -d \" > version.dat; done

real	0m0.021s
user	0m0.007s
sys	0m0.018s
```

Almost 50% speed-up ... that's worth something, right? ;-)

Reply:
As much as I like awk (the reference book is still one of the best manuals I've read to this date), I'm going to say wontfix on this. Mostly because re-enabling all the pipelines is likely more CO2 than the difference between piping to `tr` and not 🙃 (and more effort).

Reply:
I won't block the PR ;-)